website/blog/2021-09-11-sql-dateadd.md

```diff
@@ -30,23 +39,29 @@ All of them accept the same rough parameters, in slightly different syntax and o
 
 The *functions themselves* are named slightly differently, which is common across SQL dialects.
 
-### For example, in Snowflake…
+### For example, the DATEADD function in Snowflake…
 
 ```sql
 dateadd( {{ datepart }}, {{ interval }}, {{ from_date }} )
 ``` 
 
 *Hour, minute and second are supported!*
 
-### In BigQuery…
+### The DATEADD Function in Databricks
```

Comment:
I actually don't believe dateadd in dbt_utils is compatible with databricks (looking at the databricks function docs for date_add: https://docs.databricks.com/sql/language-manual/functions/date_add.html).  This is a good one to confirm w/ DX (likely Joel) though.

Reply:
@joellabes What are your thoughts? 

Some context: This post ran through how to use DATEADD in each warehouse so I decided to add to add a Databricks section to the content. We're not sure about compatibility with dbt_utils. If not, I should probably just remove the section for Databricks again.

Reply:
dbt utils itself only supports BQ/SF/RS/Postgres, but other packages can [implement overrides](https://docs.getdbt.com/reference/dbt-jinja-functions/dispatch#overriding-package-macros). That is done here for dateadd on spark (which powers Databricks, right?) https://github.com/dbt-labs/spark-utils/blob/main/macros/dbt_utils/cross_db_utils/dateadd.sql

So yes you can include Databricks in this. 

Note that these will be moving to dbt core in v1.2: https://github.com/dbt-labs/dbt-core/issues/4813

Reply:
Gotcha, so I'll make a note of that in this section of the content. Thanks Joel!!